lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -483,8 +506,13 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
     if gl_alert_count == 0 do
       serialize_single_alert_row_for_route(grouped_alerts, "Green", total_alert_count)
     else
+      gl_stop_sequences = Stop.get_gl_stop_sequences()
```

Comment:
Since `gl_stop_sequences` is only used for membership checks (as far as I can tell), you could speed things up by converting this list of lists to a list of mapsets before passing it to `alert_affects_gl_trunk_or_whole_line?/2`. The time complexity of each `in` call within `alert_affects_gl_trunk?/2` would go from `O(n)` to `O(log n)`.
```suggestion
      gl_stop_sets = Enum.map(Stop.get_gl_stop_sequences(), &MapSet.new/1)
```

Reply:
Good call. This is much better.